company_website/static/main_page/ready_to_work_with_us.sass

```diff
@@ -0,0 +1,107 @@
+@import "partials/constants"
+
+
+.ready-to-work-with-us
+    font-family: $font-family
+    font-weight: 400
+    max-width: $max-section-width
+    margin: auto
+    color: $blue
+
+    .section-title
+        text-align: center
+        text-transform: uppercase
+        color: $blue
+        font-size: 2.2rem
+        font-weight: 700
+        margin-bottom: .7rem
+
+    .row
+        max-width: 100%
+        margin: 0
+
+.background-image
```

Comment:
I see that you are using all classes which are underneath this comment in main `ready-to-work-with-us` class so please move all of them to this class to avoid any styles overwriting.

Reply:
What about background-image? I left it here with reusability in mind, since those parameters would apply to all <img> tags that will serve as a background across the page.

Reply:
Done (except for `background-image` for now)